website/data/docs-nav-data.json

```diff
@@ -52,14 +52,14 @@
         "title": "Overview",
         "path": "getting-started"
       },
-      {
-        "title": "Self-managed Boundary quick start",
-        "href": "https://developer.hashicorp.com/boundary/tutorials/oss-getting-started/"
-      },
       {
         "title": "HCP Boundary quick start",
         "href": "https://developer.hashicorp.com/boundary/tutorials/hcp-getting-started/"
       },
+      {
+        "title": "Self-managed Boundary quick start",
+        "href": "https://developer.hashicorp.com/boundary/tutorials/oss-getting-started/"
+      },
```

Comment:
I reordered this to align with the order shown in the `Get Started > Overview` page.